### ISSUE TITLE
how to build vmaf in xcode

### ISSUE THREAD

**Issue by Allen1207** [Netflix/vmaf#181]
Dear all,

I'm using mac os, and I want to study the source code of vmaf, when I open the Xcode/vmaf_xcode.xcodeproj file in Xcode and build it, the building failed with a lot of error and warnings. Therefore I want to ask is there any instruction that how to build vmaf in xcode ? Thank you.

Allen

**Comment by li-zhi** [comment]
@yuhjay 
The xcodeproj file was created a while ago. Things may have changed. Would you like to bring the project file up-to-date?

**Comment by Allen1207** [comment]
Dear li-zhi, yuhjay,

I will appreciate it if you can provide the xcode project file, or Is there any other IDE to compile, or debug the vmaf code in mac os ? Thank you.

**Comment by yuhjay** [comment]
I guess Eclipse or CLion might be an alternative to XCode.
Both of them should convert Makefile to their own projects and compile the code with clang.
Joe

**Comment by li-zhi** [comment]
@yuhjay Should we remove the xcode project file, if it is no longer maintained?

**Comment by yuhjay** [comment]
Yes, please remove the current one. I will update a new one if I can focus on this.

Thanks.
